lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -4795,6 +4795,12 @@ CompilerType SwiftASTContext::GetErrorType() {
   return {};
 }
 
+CompilerType SwiftASTContext::GetObjCObjectType() {
+  // FIXME: ClangImporter::Implementation stores this type, but it's not
+  // exposed.
+  return GetCompilerType(ConstString("$sSo8NSObjectCD"));
```

Comment:
is this symbol ABI? If so could we return the symbol in the `TypeSystemSwift` superclass instead of having both subclasses return/construct it?

Reply:
I don't want to get into the habit of hardcoding mangled names in LLDB, but we also can't call into TypeSystemSwiftTypeRef::GetObjCObjectType() here since that would cause an infinite recursion. So by having the two implementations we have at least some guarantee that the mangled name is equivalent to the demangle tree we want. It would be best to add an API to ClangImporter to return the stored type here.